kubernetes-fluentd/1.12.2-sumo-0/Dockerfile

```diff
@@ -0,0 +1,89 @@
+## Use original image to copy files from
+## ref: https://github.com/SumoLogic/sumologic-kubernetes-fluentd/blob/2d2b7693b87f9a0c175740e9674455051c709409/Dockerfile
+FROM public.ecr.aws/sumologic/kubernetes-fluentd:1.12.2-sumo-0 as sumo-image
+
+## Build RedHat compliant image for fluentd
+## ref: https://github.com/fluent/fluentd-docker-image/blob/4e0810850460c7669d7a8ad15d5a3c8ddf28652c/v1.12/debian/Dockerfile
+FROM registry.access.redhat.com/ubi8/ruby-26:1 as fluentd
+
+LABEL maintainer "Fluentd developers <fluentd@googlegroups.com>"
+LABEL Description="Fluentd docker image" Vendor="Fluent Organization" Version=1.12.2”
```

Comment:
Maybe vendor should be set to "Sumo Logic" and description corrected to "UBI based Fluentd docker image"